charts/weave-gitops/values.yaml

```diff
@@ -0,0 +1,2 @@
+images:
+  wegoApp: ghcr.io/weaveworks/wego-app:latest
```

Comment:
Probably isn't the right container image. I think we would almost always want this to be a release, but I don't know if we are going to do releases in the short/medium term? Something to thing about.